### DISCUSSION TITLE
Request: Add a Companion Reading List

### DISCUSSION THREAD

**Discussion by Mostarka** [OpenMined/courses#50] [Feedback] [answered]
Love the course so far! One request I have is to provide a companion reading list to go deeper into topics presented. For example I'd add Helen Nissenbaum 'Privacy in Context'. What are other things I should read to go deeper?

**Comment by yemikifouly** [comment]
Hello Mostarka! 🙂 

This is a great suggestion! Let me tag @iamtrask, @mcleonard and @em-blue here. I am sure they can suggest some great reading material. 

Also, Emma and Trask, is there a plan to add a reading list to the content?

**Comment by em-blue** [comment, marked as answer]
Hi @Mostarka ! :) Great question - we will be going through the course videos and adding resources for further reading on each of the topics. For now, I HIGHLY recommend these books that I really enjoyed this year:

**Privacy is Power - Carissa Veliz** (easy to read, friendly, super motivational as well)
**What Do We Know and What Should We Do About Internet Privacy?** - Paul Bernal (really easy to read and very short, great overview)
**Privacy in Context - Helen Nissenbaum** (this one is more academic, so it's less of a "light read", but it's still a great book) 
**Privacy: A Short History - David Vincent** (halfway between academic and an easy read, and really fascinating)

**Reply by Mostarka** [reply]
Amazing. Thank you Emma and Yemi!
